concent-builder/unit-tests.yml

```diff
@@ -0,0 +1,27 @@
+- hosts:
+    - concent-builder
+  vars_files:
+    - consts.yml
+    - "{{ deployment_values }}/var.yml"
+    - "{{ deployment_values }}/var-{{ cluster }}.yml"
+  tasks:
+    - become:      yes
+      become_user: "{{ shared_user }}"
+      block:
+        - name: Configure kubectl to operate on the right cluster
+          command: gcloud container clusters get-credentials \
+              "{{ gke.cluster }}"                            \
+              --project "{{ gke.project }}"                  \
+              --zone    "{{ gke.zone }}"
+
+        - name:  Run postgres container for django unit tests
+          shell: docker ps -q --filter "name=postgres" | grep -q .               \
+              || docker run --rm -d --name postgres concent-postgresql:latest
+
+        - name:    Run concent-api container and django unit tests
+          command: docker run --rm                                                                                      \
+              -v {{ build_dir }}/concent-deployment/kubernetes/build/run-unit-tests.sh:/usr/local/bin/run-unit-tests.sh \
+              --link postgres concent-concent-api:latest /usr/local/bin/run-unit-tests.sh
```

Comment:
`--link` option is deprecated. You should create a `docker network` instead and attach both containers to it using `--net` option.

See https://www.reddit.com/r/docker/comments/3smpwt/linking_with_link_will_be_deprecated_how_does/